cms/djangoapps/contentstore/tasks.py

```diff
@@ -560,3 +561,16 @@ def get_dir_for_filename(directory, filename):
                 from .views.entrance_exam import add_entrance_exam_milestone
                 add_entrance_exam_milestone(course.id, entrance_exam_chapter)
                 LOGGER.info(u'Course %s Entrance exam imported', course.id)
+
+
+@task(name='cms.djangoapps.contentstore.update_outline_from_modulestore_task')
```

Comment:
[nit] The full path would be:
```suggestion
@task(name='cms.djangoapps.contentstore.tasks.update_outline_from_modulestore_task')
```
But not a big deal at all if you'd rather leave it.